### PR TITLE
Fixed design issue

### PR DESCRIPTION
**Pull request by helloitsm3**
Made changes to issues listed in PR #667

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -107,7 +107,7 @@
     "amount": "Amount",
     "chooseToken": "Choose token",
     "enterAmount": "Enter an amount",
-    "enterTokenAmount": "Enter {{symbol}} amount",
+    "enterTokenAmount": "Enter amount",
```
[the remark this answers is not in the document]
Comment (piersss) [reply public/locales/en/translation.json:110]: And then also fix `getActionButtonTranslation` helper line 37 & 41.

**Review by piersss — CHANGES_REQUESTED** [review]
Welcome CodePerfect. Got some feedback for ya!

File: src/components/MakeWidget/MakeWidget.tsx [hunk 2]
```diff
@@ -344,11 +345,22 @@ const MakeWidget: FC = () => {
...
-        <StyledAddressInput
-          value={takerAddress}
-          onChange={setTakerAddress}
-          onInfoButtonClick={toggleShowOrderTypeInfo}
-        />
+        <Container>
+          <StyledAddressInput
+            value={takerAddress}
+            onChange={setTakerAddress}
+            takerAddressIsValid={takerAddressIsValid}
```
[the remark this answers is not in the document]
Comment (piersss) [reply src/components/MakeWidget/MakeWidget.tsx]: You have not added this yet. It still goes into error immediately.

File: src/components/MakeWidget/MakeWidget.styles.tsx [hunk 3]
```diff
@@ -62,7 +62,8 @@ export const StyledInputSection = styled(InputSection)`
 export const StyledAddressInput = styled(AddressInput)`
   margin-bottom: 1rem;
   height: 3.5rem;
-  border: ${(props) => !props?.takerAddressIsValid && "1px solid red"};
+  border: ${(props) => !props?.hasError && "1px solid red"};
```
Comment (piersss) [inline src/components/MakeWidget/MakeWidget.styles.tsx:65]: Move this styling inside the AddressInput component

**Review by piersss — CHANGES_REQUESTED** [review]
Hey Sean, please don't resolve other people's feedback next time. Thanks!

File: src/components/MakeWidget/MakeWidget.tsx [hunk 4]
```diff
@@ -344,11 +345,22 @@ const MakeWidget: FC = () => {
...
+            onInfoButtonClick={toggleShowOrderTypeInfo}
+          />
+
+          {!takerAddressIsValid && (
+            <StyledTooltip>
+              <p>
+                {takerAddress} {t("validatorErrors.invalidAddress")}
+              </p>
+            </StyledTooltip>
```
[the remark this answers is not in the document]
Comment (piersss) [reply src/components/MakeWidget/MakeWidget.tsx]: This is not fixed yet.

**Comment by helloitsm3** [comment]
Closing this PR as it's getting really messy. Will make a new PR

**Pull request closed without merging**